js/midi/plugin.webaudio.js

```diff
@@ -323,4 +317,4 @@
 			return new (window.AudioContext || window.webkitAudioContext)();
```

Comment:
Please merge this bugfix into mainstream. I was trying to play Javascript generated melodies with MIDI.js and it took me 1 hour just to find this bug, which prevents MIDI.js to work correctly, so this bug-report must be accepted.